### DISCUSSION TITLE
Talk about the smoothest transition between major releases ever...

### DISCUSSION THREAD

**Discussion by sparticvs** [sodaliterocks/sodalite#69] [Discussion]
I know this is a feature of ostree, but I just want to say that this was an incredible experience. I updated the branch and rebased - I crossed my fingers and rebooted. I knew with ostree, I could rollback if things went south and they usually do with OS Updates to new versions...but it just worked. Not only did it just work, but it felt very polished the entire time. So, thank you to the maintainer for your diligence on getting these releases out.

**Comment by electricduck** [comment]
Haha, great to know my constant testing is much appreciated 😄

**Reply by sparticvs** [reply]
Your time and efforts are very appreciated!
